<launch>

	<arg name="route_file"/>
	
	<node name="route_control"          pkg="jukovsky_ros"  type="route_control" output="screen" required="true">
		<param name="route_file" value="$(arg route_file)" type="string"/>
	</node>
	
</launch>